osu.Game/Utils/SentryLogger.cs

```diff
@@ -112,7 +102,8 @@ protected virtual void Dispose(bool isDisposing)
                 return;
 
             isDisposed = true;
-            lock (tasks) Task.WaitAll(tasks.ToArray(), 5000);
+            sentry.Dispose();
```

Comment:
I'd probably guard this for null despite `isDisposed` anyways, if this is going to be called in the finalizer - no guarantees of anything there. Better safe than sorry with these things.